README.md

```diff
@@ -31,13 +31,13 @@ We have quite a bunch of samples showing off some practical applications.
 
 ## Demo
 
-```
+```console
```

Comment:
Here is complete list of supported langs by GitHub's markdown https://github.com/github/linguist/blob/master/lib/linguist/languages.yml